components/switch/src/Switch.js

```diff
@@ -1,28 +1,14 @@
 /** @jsx jsx */
 
-import React, { createContext, useContext } from 'react';
+import React, { useState, useEffect } from 'react';
 import PropTypes from 'prop-types';
-import { jsx } from '@westpac/core';
-import { SwitchLabel } from './SwitchLabel';
-
-// ==============================
-// Context and consumer hook
-// ==============================
-
-const SwitchContext = createContext();
-
-export const useSwitchContext = () => {
-	const context = useContext(SwitchContext);
-	if (!context) {
-		throw new Error('Switch sub-components should be wrapped in a <Switch>.');
-	}
-	return context;
-};
-
+import { jsx, useBrand, useMediaQuery, merge } from '@westpac/core';
+import { VisuallyHidden } from '@westpac/a11y';
+import shortid from 'shortid';
```

Comment:
I don't think `shortid` is save to use. We should add the sequential id to `document` and just iterate it.